src/plugins/diagram-net/diagram-net.less

```diff
@@ -1,7 +1,15 @@
-.DiagramNet {}
+.DiagramNetDisplay {}
 
-.DiagramNet-img {
+.DiagramNetDisplay-image {
   display: block;
   margin: 0 auto;
   max-width: 100%;
 }
+
+.DiagramNetEditor {}
+
+.DiagramNetEditor-preview {
+  max-height: 80px;
+  max-width: 400px;
+  border: 1px solid #d9d9d9;
```

Comment:
Could we use one of our defined border colors here instead of the custom HEX code?

Reply:
Sure, this was actually `@edu-border-color-base` (probably by coincidence)